game/CMakeLists.txt

```diff
@@ -170,8 +170,9 @@ if(WIN32)
   target_link_libraries(performous wsock32 ws2_32)
 endif()
 
-target_link_libraries(performous ced)
-list(APPEND LIBS ${AUBIO_INSTALL_DIR}/lib/libaubio.a)
+target_link_libraries(performous ced aubio)
+target_link_directories(performous BEFORE PRIVATE ${AUBIO_INSTALL_DIR}/lib/)
```

Comment:
target_link_directories is only available only available from cmake 3.13 or higher.

Reply:
that is bad news, we'll have to do it in a ugly way then

Reply:
So, do we eventually need aubio to be built on windows?

Reply:
Eventually yeah, however we're not using the pre-build crosscompiled libraries